proto/cometbft/abci/v1/types.proto

```diff
@@ -261,8 +261,9 @@ message InfoResponse {
   int64 last_block_height   = 4;
   bytes last_block_app_hash = 5;
 
-  repeated uint32 lane_priorities = 6;
-  uint32 default_lane_priority = 7;
+  map<string, uint32> lane_info = 6;
```

Comment:
I would keep the name `lane_priorities` to make clear what the map is about. Otherwise I would add a comment to clarify this line.